packages/cli/test/test-utils.js

```diff
@@ -70,9 +70,16 @@ exports.givenLBProject = function(rootDir, options) {
   options = options || {};
   const sandBoxFiles = options.additionalFiles || [];
 
-  const content = {};
-  if (!options.excludeKeyword) {
-    content.keywords = ['loopback'];
+  const content = {
+    dependencies: {
+      '@loopback/core': 'X.*',
```

Comment:
Can we simplify the version spec as follows?
```suggestion
      '@loopback/core': '*',
```